modules/scrabble.js

```diff
@@ -0,0 +1,60 @@
+function scrabbleScore (str) {
+  let letterCounts = {
+    'a': 9,
+    'b': 2,
+    'c': 2,
+    'd': 4,
+    'e': 12,
+    'f': 2,
+    'g': 3,
+    'h': 2,
+    'i': 9,
+    'j': 1,
+    'k': 1,
+    'l': 4,
+    'm': 2,
+    'n': 6,
+    'o': 8,
+    'p': 2,
+    'q': 1,
+    'r': 6,
+    's': 4,
+    't': 6,
+    'u': 4,
+    'v': 2,
+    'w': 2,
+    'x': 1,
+    'y': 2,
+    'z': 1
+  }
+  var usedBlanks = ''
+  for (var letter in letterCounts) {
```

Comment:
You could use [Array.reduce()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) here if you wanted

Reply:
It might be better to loop through the string instead of the entire alphabet.
If your word is `strawberry`, you don't need to count how many times `z` occurs.

Reply:
Oh, JS has `reduce()`. That is new information.